<ui version="4.0" >
 <class>AddGeometryWidget</class>
 <widget class="QWidget" name="AddGeometryWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Primitive</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QRadioButton" name="_sphereRadioButton" >
          <property name="text" >
           <string>Sphere</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Radius</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_radiusDoubleSpinBox" />
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="_torusRadioButton" >
          <property name="text" >
           <string>Torus</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <layout class="QHBoxLayout" name="horizontalLayout_5" >
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Inner Radius</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_innerRadiusDoubleSpinBox" />
          </item>
         </layout>
        </item>
        <item row="1" column="2" >
         <layout class="QHBoxLayout" name="horizontalLayout_6" >
          <item>
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Outer Radius</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_outerRadiusDoubleSpinBox" />
          </item>
         </layout>
        </item>
        <item row="2" column="0" >
         <widget class="QRadioButton" name="_boxRadioButton" >
          <property name="text" >
           <string>Box</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QRadioButton" name="_squareRadioButton" >
          <property name="text" >
           <string>Square</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <layout class="QHBoxLayout" name="horizontalLayout_7" >
          <item>
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_boxWidth" />
          </item>
         </layout>
        </item>
        <item row="2" column="2" >
         <layout class="QHBoxLayout" name="horizontalLayout_8" >
          <item>
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_boxHeight" />
          </item>
         </layout>
        </item>
        <item row="2" column="3" >
         <layout class="QHBoxLayout" name="horizontalLayout_9" >
          <item>
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string>Depth</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_boxDepth" />
          </item>
         </layout>
        </item>
        <item row="3" column="1" >
         <layout class="QHBoxLayout" name="horizontalLayout_10" >
          <item>
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_squareWidth" />
          </item>
         </layout>
        </item>
        <item row="3" column="2" >
         <layout class="QHBoxLayout" name="horizontalLayout_11" >
          <item>
           <widget class="QLabel" name="label_13" >
            <property name="text" >
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_squareHeight" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Location</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Lat:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="_latDoubleSpinBox" >
        <property name="minimum" >
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Long:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="_longDoubleSpinBox" >
        <property name="minimum" >
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3" >
         <widget class="QLineEdit" name="_name" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QtTools::ColorButton" name="_colorButton" >
          <property name="text" >
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Transparency:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QSlider" name="_transparencySlider" >
          <property name="maximum" >
           <number>255</number>
          </property>
          <property name="value" >
           <number>255</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>gridLayoutWidget_2</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>groupBox</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTools::ColorButton</class>
   <extends>QPushButton</extends>
   <header>QtTools/ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
